src/tests/BuildWasmApps/Wasm.Build.Tests/ICUShardingTests.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Abstractions;
+using System.Globalization;
+
+#nullable enable
+
+namespace Wasm.Build.Tests
+{
+    public class ICUShardingTests : BuildTestBase
+    {
+        public ICUShardingTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buildContext)
+            : base(output, buildContext)
+        {
+        }
+
+        public static IEnumerable<object?[]> ICUShardingTestData(bool enableSharding, RunHost host)
+            => ConfigWithAOTData(false)
```

Comment:
Any reason for not testing this with AOT too?

Reply:
Would AOT make a difference here? I can enable AOT too. I guess we don't know how AOT would affect sharding.

Reply:
It will at least test the AOT build path, so we should test that here.